### PR TITLE
Add Winging It 12

### PR DESCRIPTION
**Pull request by SondraE**
## Related Issue(s)
[OddContent #165](https://github.com/orgs/oddbird/projects/13?pane=issue&itemId=69584828&issue=oddbird%7CoddFlock%7C165)

## Steps to test/reproduce
/2024/09/17/winging-it-12/
/2024/07/23/winging-it-10/

- [x] MISSING - WHAT WE COVER

### CODE REVIEW

File: content/blog/wingingit/winging-it-12.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+) }}
+
+{{ utility.main_action(
+  'Subscribe to Channel »',
+  subscribe_url
+) }}
+
+## What we cover:
+
+- When
```
Comment (stacyk) [inline content/blog/wingingit/winging-it-12.md:45]:
```suggestion
- How we plan our grids when reviewing a design
- Working with the `grid-template` shorthand property
- Making decisions about how content flows and stacks at different sizes 
- Working with `gap` and empty grid cells
- Rapid prototyping with rough markup
```

**Review by stacyk — COMMENTED** [review]
@SondraE let me know if the content I added will work

File: content/blog/wingingit/winging-it-12.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+- [Cascading Style Systems: Resilient & Maintainable CSS](https://smashingconf.com/online-workshops/workshops/modern-css-miriam-suzanne)
+
+## Demos:
+
+- [CSS Grid Design](https://www.figma.com/design/RHLs3YJE40ZAT7LiZaTQu3/Grid?node-id=2-21&node-type=frame&t=rLrO9i6kCAw1C1Ee-0)
+- [CSS Grid from Scratch](https://codepen.io/stacy/pen/ZEddQOz)
+
+## Links:
+
+-
```
Comment (stacyk) [inline content/blog/wingingit/winging-it-12.md:61]:
```suggestion
```

We didn't really have other links to my knowledge

**Comment by SondraE** [comment]
@stacyk Your additions look good to me. Thanks!

**Review by jgerigmeyer — APPROVED** [review]

**Pull request merged**